index.html

```diff
@@ -106,7 +119,7 @@ <h4 class="home-gid__item__title">Third Party Tools</h4>
                   View the tools
                 </footer>
               </a>
-            </li>
+              </li>
```

Comment:
this closing tag should be at the same indent as the opening `<li` tag (minus 2 space)